File: Data Structures & Algorithms/assign-cookies/submission-1.java
class Solution {
    public int findContentChildren(int[] g, int[] s) {
        int i = 0;
        int j = 0;
        int res = 0;
        Arrays.sort(g);
        while (j<s.length && i<g.length){
            if (s[j]>=g[i]){
                j++;
                i++;
                res++;
            }
            else{
                j++;
            }
        }
        return res ; 

    }
}

File: Data Structures & Algorithms/minimum-number-of-moves-to-seat-everyone/submission-1.py
class Solution:
    def minMovesToSeat(self, seats: List[int], students: List[int]) -> int:
        res=0
        students.sort()
        seats.sort()
        for i in range(len(seats)):
            if seats[i]!=students[i]:
                res+=abs(students[i]-seats[i])
        return res
